src/everest/simulator/__init__.py

```diff
@@ -4,7 +4,7 @@
 JOB_FAILURE = "Failed"
 
 
-DEFAULT_DATA_SUMMARY_KEYS = ("YEAR", "YEARSTCPU", "TCPUDAY", "MONTH", "DAY")
+DEFAULT_DATA_SUMMARY_KEYS = ("YEAR", "YEARS", "TCPU", "TCPUDAY", "MONTH", "DAY")
```

Comment:
wow!

Reply:
Is it worth another issue to track down why this was not caught in tests earlier?